include/zephyr/tracing/tracing.h

```diff
@@ -2376,6 +2376,65 @@
 
 /** @} */ /* end of subsys_tracing_apis_socket */
 
+/**
+ * @brief GPIO Tracing APIs
+ * @defgroup subsys_tracing_apis_gpio GPIO Tracing APIs
+ * @{
+ */
+
+/**
+ * @brief Trace a GPIO pin becoming active
+ * @param port @ref device representing the GPIO port
+ * @param pin @ref gpio_pin_t representing the pin
+ */
+#define sys_port_trace_gpio_pin_active(port, pin)
```

Comment:
As the tracing is implemented now in this PR, its not reflective of the GPIO API where `active` is not equal to `high`. The value read using `_raw()` variants of the API return `high` and `low`, not `active` and `inactive` which depend on the configuration of the pins

Reply:
This should be addressed. Please unresolve if you think it is not.